### PR TITLE
ignore missing `info/files` file

### PR DESCRIPTION
**Pull request by wolfv**
cc @beenje do you think you could deploy a change like this?

### CODE REVIEW

File: quetz/condainfo.py [hunk 1]
```diff
@@ -158,8 +158,12 @@ def _load_jsons(self, tar):
             self.paths = json.load(tar.extractfile("info/paths.json"))
         except KeyError:
             self.paths = {}
-        with tar.extractfile("info/files") as fp:
-            self.files = fp.readlines()
+
+        try:
+            with tar.extractfile("info/files") as fp:
+                self.files = fp.readlines()
+        except:
```
Comment (beenje) [inline quetz/condainfo.py:165]: Suggestion:

```suggestion
        except KeyError:
```

**Review by beenje — COMMENTED** [review]
I'll try to test that locally!

**Comment by beenje** [comment]
Tested that change locally. I uploaded a package created by rattler-build without issue. Could install it with mamba afterwards.

Looks good to me. Thanks!

**Review by pavelzw — APPROVED** [review]

**Comment by pavelzw** [comment]
Available on dockerhub now 
image

**Pull request merged**